certbot-apache/certbot_apache/configurator.py

```diff
@@ -1660,31 +1722,37 @@ def _get_proposed_addrs(self, vhost, port="80"):
 
         return redirects
 
-    def is_site_enabled(self, avail_fp):
-        """Checks to see if the given site is enabled.
-
-        .. todo:: fix hardcoded sites-enabled, check os.path.samefile
+    def parsed_in_current(self, filep):
```

Comment:
Feel free to ignore this comment if you disagree, but I think these three methods make more sense in `parser.py`.

Reply:
I do agree. These are now moved to parser.py